region/tourian/main.json

```diff
@@ -1654,15 +1773,28 @@
                   "notable": false,
                   "requires": [
                     {"or": [
-                      "ScrewAttack",
-                      "Ice"
+                      "Ice",
+                      {"and": [
+                        "ScrewAttack",
+                        "canDodgeWhileShooting"
+                      ]}
                     ]}
                   ]
                 },
                 {
-                  "name": "Harder Evade",
-                  "notable": false,
-                  "requires": ["canMetroidAvoid"]
+                  "name": "Metroid Room 4 Bottom Metroid Avoid",
+                  "notable": true,
+                  "requires": [
+                    "canMetroidAvoid",
+                    "canDodgeWhileShooting",
+                    "canCarefulJump"
```

Comment:
I would think this would be `tricky jump`. Is it a similar difficulty when coming in through any location on the door? I thought there were some ways (far left?) that were more difficult

Reply:
carefuljump is more for avoiding rinkas.  I didn't look at the room entries as closely as in the past because I thought there was a common room entry method of run right + shoot left.  I can retest it

Reply:
Different room entries are still a bit different but the idea is the same.  Shoot the metroids to keep them slowed.  And waiting on the far right side of the door is fine for bringing down mid metroid safely.